addons/fleet/views/fleet_vehicle_model_views.xml

```diff
@@ -37,16 +44,14 @@
                                 <group string="Model" invisible="vehicle_type != 'car'">
                                     <field name="seats"/>
                                     <field name="doors"/>
-                                    <field name="color"/>
+                                    <field name="color" invisible="vehicle_type == 'car'"/>
```

Comment:
According to the spec color should not be in the view no ?

Reply:
Geoffrey's review : Color on model should be hidden if vehicle type = car ; I think I can remove the field :thinking: